LEAF_Request_Portal/templates/print_form.tpl

```diff
@@ -106,70 +123,108 @@
 <script type="text/javascript" src="js/functions/toggleZoom.js"></script>
 <script type="text/javascript" src="../libs/js/LEAF/sensitiveIndicator.js"></script>
 <script type="text/javascript">
+$(document).ready(function() {
+  $(window).keydown(function(event){
+    if(event.keyCode == 13) {
+      event.preventDefault();
+      submitNote(<!--{$recordID|strip_tags}-->);
+      return false;
+    }
+  });
+});
+
 var currIndicatorID;
 var currSeries;
 var recordID = <!--{$recordID|strip_tags}-->;
 var serviceID = <!--{$serviceID|strip_tags}-->;
 var CSRFToken = '<!--{$CSRFToken}-->';
 var formPrintConditions = {};
 function doSubmit(recordID) {
-	$('#submitControl').empty().html('<img src="./images/indicator.gif" />Submitting...');
-	$.ajax({
-		type: 'POST',
-		url: "./api/form/" + recordID + "/submit",
-		data: {CSRFToken: '<!--{$CSRFToken}-->'},
-		success: function(response) {
+    $('#submitControl').empty().html('<img src="./images/indicator.gif" />Submitting...');
+    $.ajax({
+        type: 'POST',
+        url: "./api/form/" + recordID + "/submit",
+        data: {CSRFToken: '<!--{$CSRFToken}-->'},
+        success: function(response) {
             if(response.errors.length == 0) {
                 $('#submitStatus').text('Request submmited');
                 $('#submitControl').empty().html('Submitted');
                 $('#submitContent').hide('blind', 500);
                 workflow.getWorkflow(recordID);
             }
             else {
-            	var errors = '';
-            	for(var i in response.errors) {
-            		errors += response.errors[i] + '<br />';
-            	}
+                var errors = '';
+                for(var i in response.errors) {
+                    errors += response.errors[i] + '<br />';
+                }
                 $('#submitControl').empty().html('Error: ' + errors);
                 $('#submitStatus').text('Request can not be submmited');
             }
-		}
-	});
+        }
+    });
+}
+
+function submitNote(recordID){
```

Comment:
default value.  Actually, may want to check if it's valid before kicking off an ajax call.  Also, fail catch.